FlowCrypt/Controllers/Compose/ComposeViewController.swift

```diff
@@ -144,27 +195,53 @@ extension ComposeViewController {
     private func encryptAndSendMessage() -> Promise<Bool> {
         Promise<Bool> { [weak self] () -> Bool in
             guard let self = self else { return false }
-            guard let email = self.contextToSend.resipient, let text = self.contextToSend.message else {
+
+            let recipients = self.contextToSend.recipients
+
+            guard recipients.isNotEmpty else {
+                assertionFailure("Recipients should not be empty. Fail in checking");
+                return false
+            }
+
+            guard let text = self.contextToSend.message else {
                 assertionFailure("Text and Email should not be nil at this point. Fail in checking");
                 return false
             }
-            let subject = self.input.subjectReplyTitle ?? self.contextToSend.subject ?? "(no subject)"
-            let lookupRes = try await(self.attesterApi.lookupEmail(email: email))
-            guard let recipientPubkey = lookupRes.armored else {
-                self.showAlert(message: "compose_no_pub_recipient".localized)
+
+            let subject = self.input.subjectReplyTitle
+                ?? self.contextToSend.subject
+                ?? "(no subject)"
+
+
+            let lookup = recipients.map {
+                self.attesterApi.lookupEmail(email: $0.email)
+            }
+
+            let lookupRes = try await(all(lookup))
+            let allRecipientPubs = lookupRes.compactMap { $0.armored }
+
+            guard allRecipientPubs.count == recipients.count else {
+                let message = recipients.count == 1
+                    ? "compose_no_pub_recipient".localized
+                    : "compose_no_pub_multiple".localized
+                self.showAlert(message: message)
                 return false
             }
-            guard let myPubkey = self.dataManager.publicKey() else {
+
+            guard let myPubKey = self.dataManager.publicKey() else {
                 self.showAlert(message: "compose_no_pub_sender".localized)
                 return false
             }
+
             let encrypted = self.encryptMsg(
-                pubkeys: [myPubkey, recipientPubkey],
+                pubkeys: allRecipientPubs + [myPubKey],
                 subject: subject,
                 message: text,
-                email: email
+                email: recipients.map { $0.email }
```

Comment:
here `email:` could be renamed to `to:` since it's now an array. Also, in the future we may add `cc:` and `bcc:` so it makes sense to distinguish it now.